src/main/java/com/volasoftware/tinder/utility/BotGenerator.java

```diff
@@ -34,4 +33,15 @@ private static User createBot() {
 
         return botUser;
     }
+
+    private static Location createLocationForUser(User user) {
+        Random random = new Random();
+        Location location = new Location();
+
+        location.setUser(user);
+        location.setLatitude(random.nextDouble());
```

Comment:
random.nextDouble should there be some kind of a range, maybe upper and lower bound for the return value?